create_db.sh

```diff
@@ -1,9 +1,11 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I'll revert this PR - I guess this isn't the place to decide what common_startup.sh should be.